satpy/readers/landsea_nc.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017-2019 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Land-sea mask from MODIS format reader.
+
+References:
+   - The land sea mask from MODIS, documentation: https://www.ghrsst.org/ghrsst-data-services/services/
```

Comment:
I don't see any documentation for the land/sea mask here, just links to SST data sources ?

Reply:
yes, it is only a download link: https://www.ghrsst.org/ghrsst-data-services/tools/
I corrected that.